packages/create-react-app/createReactApp.js

```diff
@@ -49,7 +49,7 @@ const url = require('url');
 const hyperquest = require('hyperquest');
 const envinfo = require('envinfo');
 const os = require('os');
-
+const monorepo = require('react-dev-utils/monorepo');
```

Comment:
Let's use a name like`WorkspaceUtils` so it's clearer it's a bunch of functions.